src/kibana/components/doc_table/components/table_row.js

```diff
@@ -104,10 +105,10 @@ define(function (require) {
           }
 
           $scope.columns.forEach(function (column) {
-            newHtmls.push(cellTemplate({
+            newHtmls.push(noWhiteSpace(cellTemplate({
```

Comment:
I'd say that the _source field shouldn't have significant white-space. It if does then a couple new lines will hide the rest of the fields. What do you thing @rashidkpc?